### ISSUE TITLE
No nice pic shown, when link is shared

### ISSUE THREAD

**Issue by fzesch** [fzesch/lagesonum#50]
When our site is shared, e.g. on facebook or twitter, we don't have meaningful picture as a symbol. On the other hand, pictures would increase traffic, what might be undesired for those on a small 3G-plan.

**Comment by pitpalme** [comment]
https://developers.facebook.com/docs/sharing/best-practices

Maybe

``` html
<meta property="og:image" content="http://lagesonum/static/logo-100x100.jpg" />
```

is enough? Eventually there's something similar for Twitter.

At least it wouldn't significantly increase traffic for users.

**Comment by fzesch** [comment]
Good idea! Maybe we could even use more open graph lingo:

``` html
<meta property="og:title" content="The best site">
<meta property="og:image" content="link_to_image">
<meta property="og:description" content="description goes here">
```

**Comment by fzesch** [comment]
Twitter has [Twitter cards](https://dev.twitter.com/cards/overview)

**Comment by fzesch** [comment]
fixed 6a4de742223b6a7a421223fad4032067342f25eb
